tools/buildpkg.sh

```diff
@@ -122,21 +119,22 @@ git submodule -q update --init --recursive >/dev/null || exit 1
 echo "Done"
 
 echo "Installing composer dependencies:"
+curl -sS https://getcomposer.org/installer | php
 echo -n " - lib/pkp ... "
 cd lib/pkp
-composer.phar update --no-dev
+php ../../composer.phar update --no-dev
```

Comment:
I think this should be install actually. It fixed (some) problems for me.

Reply:
ok